app/src/main/java/com/iic/shopingo/ui/activities/RequestDetails.java

```diff
@@ -0,0 +1,78 @@
+package com.iic.shopingo.ui.activities;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.PersistableBundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import com.iic.shopingo.PriceHelper;
+import com.iic.shopingo.R;
+import com.iic.shopingo.data.Request;
+
+/**
+ * Created by asafg on 03/03/15.
+ */
+public class RequestDetails extends Activity {
+  public static final String EXTRA_REQUEST = "request";
+
+  public static final int RESULT_NONE = 0;
+  public static final int RESULT_ACCEPT = 1;
+  public static final int RESULT_REJECT = 2;
```

Comment:
He wrote an enum for states, which is not the same as an enum for results.
These result (while they have similar names to other states in the system), belong in this class, because these are the results that this specific class returns.
Also, it's not an enum because android needs ints. I don't think there's an advantage to enums in this specific case.